packages/docusaurus-remark-plugin-npm2yarn/src/__tests__/index.test.ts

```diff
@@ -55,4 +58,24 @@ describe('npm2yarn plugin', () => {
 
     expect(result).toMatchSnapshot();
   });
+
+  it('work with yarn converter', async () => {
+    const result = await processFixture('plugin', {converters: ['yarn']});
+
+    expect(result).toMatchSnapshot();
+  });
```

Comment:
would appreciate it if you could add a test case for: `npm run xxx -- --arg` vs `yarn xxx --arg`

It's likely to be solved in npm-to-yarn but no test means I'm not 100% sure we can. rely on it 🤪